### PR TITLE
Natasha/fixes withlarge title

### PR DESCRIPTION
**Pull request by natashaarmbrust**
This is the opposite of the branch name (idk why I named it "with large title") --> i.e. all this branch is all the fixes before adding the large titles. Please review this PR and not the `natasha/quickFixes`#386  PR

### CODE REVIEW

File: Podcast/PlayerRateSlider.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    var values = PlayerRate.values()
+
+    init() {
+        super.init(frame: .zero)
+        trackHeight = 3
+        trackCircleRadius = 4
+        sliderCircleRadius = 10
+        isDotsInteractionEnabled = true
+        trackColor = .paleGrey
+        sliderCircleColor = .paleGrey
```
Comment (mindylou) [inline Podcast/PlayerRateSlider.swift]: can you make the big circle color `sea`? just so it's easier to recognize which value you're currently on

File: Podcast/UserSettings.swift [hunk 2]
```diff
@@ -82,6 +82,32 @@ class ChangeUsernameSettingsPageViewController: SettingsPageViewController {
...
+        super.init(nibName: nil, bundle: nil)
+        sections = setupSettings()
+        showSave = true
+        title = "Custom Player Settings"
+        returnOnSave = false
+    }
+
+    func setupSettings() -> [SettingsSection] {
+        return [
+            SettingsSection(id: "palyer_rate_section", items: [
```
Comment (mindylou) [inline Podcast/UserSettings.swift]: `player_rate_section`*

File: Podcast/UserSettings.swift [hunk 3]
```diff
@@ -82,6 +82,32 @@ class ChangeUsernameSettingsPageViewController: SettingsPageViewController {
...
+                SettingsField(id: "plater_rate", title: "Default Rate:", saveFunction: { rate in
```
Comment (mindylou) [inline Podcast/UserSettings.swift]: `player_rate`* lol

**Review by mindylou — APPROVED** [review]
LGTM after the small changes!
I think eventually we should add the player rate settings to the player itself but we should wait until we figure out all the settings/designs for that

File: Podcast/PlayerRateSlider.swift (hunk 1 above)
Comment (natashaarmbrust) [reply Podcast/PlayerRateSlider.swift]: yup good idea!

File: Podcast/UserSettings.swift (hunk 2 above)
Comment (natashaarmbrust) [reply Podcast/UserSettings.swift]: lmao

**Pull request merged**